### PR TITLE
added TOSHI - HZN - BALD for BAS

### PR DESCRIPTION
**Pull request by H3xept**
Tokens highlighted here: https://lifi.atlassian.net/browse/LF-5057

### CODE REVIEW

File: tokens/BAS.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+[{
+	"name": "Toshi Token",
+	"address": "0xF136D7b0B7AE5b86D21E7B78DFA95375a7360f19",
```
Comment (myz1237) [inline tokens/BAS.json:3]: https://basescan.org/address/0xF136D7b0B7AE5b86D21E7B78DFA95375a7360f19
Incorrect contract addr?

Comment (H3xept) [reply tokens/BAS.json:3]: Oops yes!

File: tokens/BAS.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+[{
+	"name": "Toshi Token",
+	"address": "0x8544FE9D190fD7EC52860abBf45088E81Ee24a8c",
+	"symbol": "TOSHI",
+	"decimals": 18,
+	"chainId": 8453,
+	"logoURI": "https://static.debank.com/image/eth_token/logo_url/0xf136d7b0b7ae5b86d21e7b78dfa95375a7360f19/7727b7001e9a63b857bf48afae76da99.png"
```
Comment (myz1237) [inline tokens/BAS.json:7]: Seems that the logo should be changed? Since https://basescan.org/token/images/toshibase2_32.png is different from current one

Comment (H3xept) [reply tokens/BAS.json]: We got the address from debank, I updated it but both should work.

**Review by myz1237 — APPROVED** [review]

**Pull request merged**